packages/app/cypress/e2e/runner/support/spec-loader.ts

```diff
@@ -3,11 +3,11 @@ export const shouldHaveTestResults = ({ passCount, failCount, pendingCount }) =>
   failCount = failCount || '--'
```

Comment:
Had to increase some timeouts here to work around potentially-long bootstrap times for these tests when run under CT due to bundler overhead